src/Nox.Generator/Application/Commands/CreateCommand.template.cs

```diff
@@ -1,4 +1,3 @@
-﻿{{- relatedEntities = entity.Relationships | array.map "Entity" }}
```

Comment:
so were do we compute the related entities?

Reply:
we compute related entites in CreateCommandGenerator.cs lines between 28 to 33